doc/level1.rst

```diff
@@ -677,6 +677,10 @@ each of which:
   referenced by ``addr``. See also :ref:`preliminary recommendation`
   for how this key is selected.
 
+- SHOULD contain only a single user id in the key material from the
+  ``keydata`` attribute, which matches the ``addr`` attribute. This
+  avoids leaking unrelated identities between recipients.
```

Comment:
huh? why suddenly tie the user id to the addr? we say in the definition of the header format: "The content of the user id packet is only decorative.".

Reply:
i think the pr belongs to the irc chat from yesterday. 
imu, it's not about to create a user id packet _belonging_ to addr; it's more about stripping all user id packets _not_ belonging to addr (eg. if the key is no autocrypt key and has legacy identities, ancient data and so on).

Reply:
then i think this third "SHOULD" point should just be a clarification to the second MUST eg:

 - MUST include the ``keydata`` attribute which MUST contain the
   same public key which is used to encrypt the mail to the recipient
   referenced by ``addr``.     See also  :ref:`preliminary recommendation`
   for how this key is selected. Note that for non-Autocrypt keys, only the one user id 
   which relates to ``addr`` SHOULD be contained in  ``keydata``.

Reply:
You're right, I had totally forgotten about the decorative part. It's indeed only about stripping out packets that aren't relevant, if more than one is available. I worded this differently.

I added this I think keeping this as a separate point is reasonable since it's only relevant if the key is somehow merged or comes from an external source. I also added a more explicit note about that